session.rst

```diff
@@ -237,43 +695,926 @@ your ``Session`` object with the default ``AttributeBag`` by the ``NamespacedAtt
         // config/services.php
         namespace Symfony\Component\DependencyInjection\Loader\Configurator;
 
-        use Symfony\Component\HttpFoundation\Session\Attribute\NamespacedAttributeBag;
-        use Symfony\Component\HttpFoundation\Session\Session;
+        use Symfony\Component\HttpFoundation\Session\Storage\Handler\PdoSessionHandler;
 
-        return function(ContainerConfigurator $containerConfigurator) {
-            $services = $containerConfigurator->services();
+        return static function (ContainerConfigurator $container) {
+            $services = $configurator->services();
```

Comment:
The variable is called `$container` the line above.
I would change this one, but I know there may still be discussions about this.
(this error seems present in some other blocks)